### ISSUE TITLE
Experiment reproduction

### ISSUE THREAD

**Issue by pglez82** [cml-cs-iastate/CyText#1]
Dear @leiqi,

I am interested in your DQN paper and I am trying to reproduce the experiments. It's giving me a lot of problems as you do not indicate which library versions you are using (tensorflow, keras, numpy, etc). Also, It is not clear to me how you load the data. I am trying to load the imdb dataset but it is not clear how you load the labels. Could you upload the data files that you used for executing your code.

Thanks!
